.github/workflows/tests-unit-php.yml

```diff
@@ -136,15 +136,15 @@ jobs:
         run: |
           if [[ $WP_VERSION == "5.7" && $PHP_VERSION == "7.2" ]]; then
             echo "Installing PHPUnit 7.5.x"
-            composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/phpunit:^7.5 --with-dependencies
+            composer require --no-interaction --no-scripts phpunit/phpunit:^7.5 --with-dependencies
           elif [[ $PHP_VERSION == "7.2" ]]; then
             echo "Installing PHPUnit 8.5.x"
-            composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/phpunit:^8.5 --with-dependencies
+            composer require --no-interaction --no-scripts phpunit/phpunit:^8.5 --with-dependencies
             echo "Downgrading phpunit/php-token-stream because the latest version requires PHP 7.3"
-            composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/php-token-stream:^3.1.3 --with-dependencies
+            composer require --no-interaction --no-scripts phpunit/php-token-stream:^3.1.3 --with-dependencies
           elif [[ $WP_VERSION == "latest" || $WP_VERSION == "trunk" || $PHP_VERSION == "7.4" ]]; then
             echo "Installing latest version of PHPUnit"
-            composer update --ignore-platform-reqs --no-interaction --no-scripts yoast/phpunit-polyfills --with-dependencies
```

Comment:
I think these changes need to be undone, because it prevents actually installing different PHPUnit versions.

On WP latest & trunk it's supposed to install PHPUnit 9 (the latest version of PHPUnit), but it's not happening because of the platform requirement, thus still running with PHPunit 8.